cms/articles/tests/test_models.py

```diff
@@ -233,6 +233,10 @@ def test_cite_this_page_is_not_shown_when_unticked(self):
         response = self.client.get(self.basic_page_url)
         self.assertNotContains(response, expected)
 
+    def test_breadcrumb_doesnt_containt_series_url(self):
+        response = self.client.get(self.basic_page_url)
+        self.assertContains(response, self.basic_page.get_parent().url)
```

Comment:
Ah, of course this fails. The Statistical article URL contains the Article Series URL. And we have 2 (the Open Graph tag) and "view previous releases" link.

so the check should be something like

```python
    def test_breadcrumb_doesnt_contain_series_url(self):
        response = self.client.get(self.basic_page_url)

        parent = self.basic_page.get_parent()
        self.assertNotContains(
            response,
            f'<a class="ons-breadcrumbs__link" href="{parent.url}">{parent.title}</a>',
            html=True,
        )
        ancestor = parent.get_parent()
        self.assertContains(
            response,
            f'<a class="ons-breadcrumbs__link" href="{ancestor.url}">{ancestor.title}</a>',
            html=True,
        )
```

Reply:
Thanks Dan was trying to get the negative test